core/domain/question_domain.py

```diff
@@ -240,20 +267,38 @@ def update_question_data(self, question_data):
         """
         self.question_data = question_data
 
+    def update_question_status(self, status):
+        """Updates the question data of the question.
+
+        Args:
+            status: str. The status of the question.
+        """
+        self.status = status
+
 
 class QuestionSummary(object):
     """Domain object for Question Summary.
     """
-    def __init__(self, question_id, question_content):
+    def __init__(
+            self, question_id, creator_id, question_html_data,
+            question_model_last_updated=None, question_model_created_on=None):
         """Constructs a Question Summary domain object.
 
         Args:
             question_id: str. The ID of the question.
-            question_content: str. The static HTML of the question shown to
+            creator_id: str. The user ID of the creator of the question.
+            question_model_last_updated: datetime.datetime. Date and time
```

Comment:
Order of args does not match order in docstring.

@apb7 @kevinlee12 we should catch this in lint checks.

Reply:
Added to #3905 under the Python section. Thanks!

Reply:
Done